scripts/baseline_view.py

```diff
@@ -140,7 +141,7 @@ def _baseline_callback_ecef(self, data):
     return
 
   def iar_state_callback(self, data):
-    self.num_hyps = struct.unpack('<I', data)[0]
+    self.num_hyps = struct.unpack('<I', data.payload)[0]
```

Comment:
... with updating the contract, this could be something like: 

```python
def iar_state_callback(self, sbp_msg):
  self.num_hyps = MsgIarState(sbp_msg).num_hyps
```